iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/IcebergRestUtils.java

```diff
@@ -71,4 +80,48 @@ public static Instant calculateNewTimestamp(Instant currentTimestamp, int hours)
     }
     return nextHourDateTime.atZone(ZoneId.systemDefault()).toInstant();
   }
+
+  public static String getCatalogName(String rawPrefix) {
+    String prefix = shelling(rawPrefix);
+    Preconditions.checkArgument(
+        !IcebergConstants.GRAVITINO_DEFAULT_CATALOG.equals(prefix),
+        String.format("%s is conflict with reserved key, please replace it", prefix));
+    if (StringUtils.isBlank(prefix)) {
+      return IcebergConstants.GRAVITINO_DEFAULT_CATALOG;
+    }
+    return prefix;
+  }
+
+  public static NameIdentifier getGravitinoNameIdentifier(
+      String catalogName, TableIdentifier icebergIdentifier) {
+    Stream<String> catalogNS =
+        Stream.concat(
+            Stream.of(catalogName), Arrays.stream(icebergIdentifier.namespace().levels()));
+    String[] catalogNSTable =
+        Stream.concat(catalogNS, Stream.of(icebergIdentifier.name())).toArray(String[]::new);
+    return NameIdentifier.of(catalogNSTable);
+  }
+
+  // remove the last '/' from the prefix, for example transform 'iceberg_catalog/' to
+  // 'iceberg_catalog'
+  private static String shelling(String rawPrefix) {
```

Comment:
What's the meaning of `shelling`, it is hard to understand, can we use a better name?